src/transformers/file_utils.py

```diff
@@ -1107,8 +1143,8 @@ def get_from_cache(
             incomplete_path = cache_path + ".incomplete"
 
             @contextmanager
-            def _resumable_file_manager():
-                with open(incomplete_path, "a+b") as f:
+            def _resumable_file_manager() -> "io.BufferedWriter":
```

Comment:
AFAIK contextmanager's should return an Iterator. Also see [this SO question](https://stackoverflow.com/questions/49733699/python-type-hints-and-context-managers).